docs/guides/deployment/digitalocean.rst

```diff
@@ -181,35 +199,47 @@ Set the security policy to strict.
 That's it! You can now start using the EdgeDB instance located at
 ``edgedb://$IP``.
 
-
 .. _ref_guide_deployment_digitalocean_link:
 
-Create a local link to the new EdgeDB instance
-==============================================
+Get your instance's DSN
+=======================
 
-To access the EdgeDB instance you've just provisioned on DigitalOcean from your
-local machine run the following command.
+Let's construct your instance's DSN (also known as a "connection string").
```

Comment:
Is this duplicated? Appears to be the exact same documentation as above for constructing a DSN.

Reply:
Relevant to lines 215-223 in the file change.